.clabot

```diff
@@ -0,0 +1,6 @@
+{
+  "contributors": [],
+  "message": "OpenCannabis requires contributors to sign a _Contributor License Agreement_, and we don\"t have one on file for you! In order for us to review and merge your code, please contact @OpenCannabis/editors to get yourself added.",
+  "label": "CLA: Ready",
+  "recheckComment": "clacheck"
```

Comment:
pretty straight forward, it's here: https://colineberhardt.github.io/cla-bot/#installing-cla-bot